src/main/java/org/baeldung/Application.java

```diff
@@ -1,4 +1,4 @@
-package org.baeldung.spring;
```

Comment:
@eugenp , as told in above review only https://github.com/Baeldung/spring-security-registration/pull/24/commits/0c3b31bce4e9b1dfd67446d7703e49a3ba04287b this matched PR name